views/peeps/index.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <style>body{background-color: #66b3ff;}
+</style>
+</head>
+
+<form action="/peeps/new" method="get">
+     <button type="submit">New Peep</button>
+ </form>
+
+<ul>
+  <% @peeps.reverse.each do |peep| %>
```

Comment:
Great to learn this is how you reversed the order of the posts that showed here.

Reply:
Thanks, think you can also use the reverse_each method here.